tutorials/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -342,6 +342,29 @@ Example:
 
     Input.Singleton.JoyConnectionChanged += Input_JoyConnectionChanged;
 
+Float
+-----
+
+Some engine functions return or have as input parameters values of the built-in
+:ref:`float <class_float>` type. This type is a 64-bit double-precision floating-point
+number, equivalent to ``double`` in C# (`System.Double` for .Net).
```

Comment:
The [.NET brand guidelines](https://github.com/dotnet/brand/blob/main/dotnet-brand-guidelines.pdf) say the name is all uppercase.

```suggestion
number, equivalent to ``double`` in C# (`System.Double` for .NET).
```